tests/functional/ovndbcluster_controller_test.go

```diff
@@ -226,6 +227,51 @@ var _ = Describe("OVNDBCluster controller", func() {
 			DeferCleanup(th.DeleteInstance, instance)
 		})
 
+		It("should create services", func() {
+			internalAPINADName := types.NamespacedName{Namespace: namespace, Name: "internalapi"}
+			nad := th.CreateNetworkAttachmentDefinition(internalAPINADName)
+			DeferCleanup(th.DeleteInstance, nad)
+
+			statefulSetName := types.NamespacedName{
+				Namespace: namespace,
+				Name:      "ovsdbserver-sb",
+			}
+			th.SimulateStatefulSetReplicaReadyWithPods(
+				statefulSetName,
+				map[string][]string{namespace + "/internalapi": {"10.0.0.1"}},
+			)
+			service := types.NamespacedName{
+				Namespace: OVNDBClusterName.Namespace,
+				Name:      "ovsdbserver-sb",
+			}
+			// Even with the Eventually OVN takes to long to reconcile and create both services
+			// with this extra 2 seconds by the time the check is done, services will be already created
+			time.Sleep(2 * time.Second)
```

Comment:
this doesn't seem needed, Can you try dropping this workaround + fixing operator-lint issues

Reply:
Done